index.js

```diff
@@ -149,24 +149,41 @@ class ServerlessCustomDomain {
       let nameLength = 0;
       // The arn of the choosen certificate
       let certificateArn;
+      // The certificate name
+      let certificateName = this.serverless.service.custom.customDomain.certificateName;
+
+      // Checks if a certificate name is given
+      if (certificateName.trim() !== '') {
```

Comment:
What if `certificateName` is missing from the config. Does this error out since `certificateName` will be `undefined`